tank_battalion/shell.pde

```diff
@@ -43,8 +43,26 @@ class Shell {
     
     if (physics_manager.check_collision(x, y, SIZE, SIZE, collider_id, layer_mask))
     {
+      if ((layer_mask & PLAYER_LAYER) == 0){
```

Comment:
I agree, it makes no sense but when I tried 
```((layer_mask & ENEMY_LAYER) == 1)``` 
it didn't work for some reason...

Reply:
@robkruger In that case you needed to use `((layer_mask & ENEMY_LAYER) > 0)` 
because the outcome can be anything greater than zero. That's just how layer masks work because you can have multiple layers as outcome.

Reply:
But I fixed it in another way that doesn't even need that system